src/Components/Navbar/UserNavbar.js

```diff
@@ -76,11 +98,37 @@ export default function UserNavBar(props) {
     }
   ];
   const unauthedRoutes = [{ title: 'Events', route: '/events' }];
-
   const toggler = () => {
     setMenuIsOpen(!menuIsOpen);
   };
 
+  function createCookie() {
+    if (document.body.className === 'light') {
+      document.body.className = 'dark';
+      setDarkTheme(true);
+      enableDarkMode();
+      cook.set('thisCookie', 'true', { path: '/' });
+    } else {
+      document.body.className = 'light';
+      setDarkTheme(false);
+      disableDarkMode();
+      cook.set('thisCookie', 'false', { path: '/' });
+    }
+  }
+
+  function keepState() {
+    if (cook.get('thisCookie') === 'true') {
+      setDarkTheme(true);
+    } else {
+      setDarkTheme(false);
+    }
+  }
+
+  useEffect(() => {
+    keepState();
```

Comment:
perhaps rename this to
```suggestion
    checkIfDarkThemeActive();
```